<element name="surface" required="0">
  <element name="bounce" required="0">
    <attribute name="restitution_coefficient" type="double" default="0" required="0"/>
    <attribute name="threshold" type="double" default="100000" required="0"/>
  </element> <!-- End Bounce -->

  <element name="friction" required="0">
    <element name="ode" required="0">
      <attribute name="mu" type="double" default="-1" required="0"/>
      <attribute name="mu2" type="double" default="-1" required="0"/>
      <attribute name="fdir1" type="vector3" default="0 0 0" required="0"/>
      <attribute name="slip1" type="double" default="0.0" required="0"/>
      <attribute name="slip2" type="double" default="0.0" required="0"/>
    </element> <!-- End ODE -->
  </element> <!-- End Friction -->

  <element name="contact" required="0">
    <element name="ode" required="0">
      <attribute name="soft_cfm" type="double" default="0" required="0"/>
      <attribute name="soft_erp" type="double" default="0.2" required="0"/>
      <attribute name="kp" type="double" default="1000000000000.0" required="0"/>
      <attribute name="kd" type="double" default="1.0" required="0"/>
      <attribute name="max_vel" type="double" default="0.01" required="0"/>
      <attribute name="min_depth" type="double" default="0" required="0"/>
    </element> <!-- End ODE -->
  </element> <!-- End Contact -->
</element> <!-- End Surface -->
